src/app/Components/ArtsyWebView.tsx

```diff
@@ -280,21 +282,27 @@ export const ArtsyWebView = forwardRef<
       }
     }
 
+    useEffect(() => {
+      getUserAgent().then((agent) => setUserAgent(agent))
+    }, [])
+
     return (
       <Flex flex={1}>
         <WebView
+          enableApplePay={true}
           ref={innerRef}
           // sharedCookiesEnabled is required on iOS for the user to be implicitly logged into force/prediction
           // on android it works without it
           sharedCookiesEnabled
+          webviewDebuggingEnabled
```

Comment:
is this intentional? I was assuming we would do something like this
```suggestion
          webviewDebuggingEnabled={__DEV__}
```

Reply:
This PR is still in progress 😅

Reply:
oups, so sorry